plotnine/geoms/geom_point.py

```diff
@@ -31,14 +31,13 @@ def draw_panel(self, data, panel_params, coord, ax, **params):
         """
         self.draw_group(data, panel_params, coord, ax, **params)
 
-    @staticmethod
-    def draw_group(data, panel_params, coord, ax, **params):
+    @classmethod
```

Comment:
For uniformity and since it is backwards compatible, change all `geom.draw_group` to classmethod.